index.html

```diff
@@ -1017,6 +1016,120 @@ <h3>Interface Definition</h3>
+  unsigned long long bytesSent;
+  unsigned long long packetsSent;
+  unsigned long numOutgoingStreamsCreated;
+  unsigned long numIncomingStreamsCreated;
+  unsigned long long bytesReceived;
+  unsigned long long packetsReceived;
+  unsigned long long minRttUs;
+  unsigned long estimatedBandwidthBps;
+  unsigned long numReceivedDatagramsDropped;
+};</pre>
+        <section>
+          <h2>Dictionary <a class="idlType">QuicTransportStats</a> Members</h2>
+          <dl data-link-for="QuicTransportStats" data-dfn-for="QuicTransportStats" class=
+          "dictionary-members">
+            <dt><dfn><code>timestamp</code></dfn> of type <span class=
+            "idlMemberType"><a>DOMHighResTimeStamp</a></span></dt>
+            <dd>
+              <p>The <code>timestamp</code> for when the stats are gathered, relative to the
+              UNIX epoch (Jan 1, 1970, UTC)</p>
+            </dd>
+            <dt><dfn><code>bytesSent</code></dfn> of type <span class=
+            "idlMemberType"><a>unsigned long long</a></span></dt>
+            <dd>
+              <p>The amount of bytes sent on the QUIC connection, including
+              retransmissions. Does not include UDP/IP/ethernet framing.</p>
+            </dd>
+            <dt><dfn><code>packetsSent</code></dfn> of type <span class=
+            "idlMemberType"><a>unsigned long long</a></span></dt>
+            <dd>
+              <p>The amount of packets sent</code> on the QUIC connection, including
+              retransmissions.</p>
+            </dd>
+            <dt><dfn><code>numOutgoingStreamsCreated</code></dfn> of type <span class=
+            "idlMemberType"><a>unsigned long</a></span></dt>
+            <dd>
+              <p>The amount of outgoing QUIC streams created on the QUIC connection.</p>
+            </dd>
+            <dt><dfn><code>numIncomingStreamsCreated</code></dfn> of type <span class=
+            "idlMemberType"><a>unsigned long</a></span></dt>
+            <dd>
+              <p>The amount of incoming QUIC streams created on the QUIC connection.</p>
+            </dd>
+            <dt><dfn><code>bytesReceived</code></dfn> of type <span class=
+            "idlMemberType"><a>unsigned long long</a></span></dt>
+            <dd>
+              <p>The amount of total bytes received on the QUIC connection, including
+              duplicate data for streams. Does not include UDP/IP/ethernet framing.</p>
+            </dd>
+            <dt><dfn><code>packetsReceived</code></dfn> of type <span class=
+            "idlMemberType"><a>unsigned long long</a></span></dt>
+            <dd>
+              <p>The amount of total packets received on the QUIC connection, including
+              packets that were not processable.</p>
+            </dd>
+            <dt><dfn><code>minRttUs</code></dfn> of type <span class=
+            "idlMemberType"><a>unsigned long long</a></span></dt>
+            <dd>
+              <p>The minimum RTT in microseconds.</p>
```

Comment:
What time units we use elsewhere in the spec?  `timestamp` is a `DOMHighResTimeStamp`, which is milliseconds represented as `double`.  We should try to be consistent.